doc/bucket-replication.md

```diff
@@ -0,0 +1,82 @@
+[NooBaa Operator](../README.md) /
+
+# Bucket Replication
+Bucket replication is a NooBaa feature that allows a user to set a replication policy for all or some objects. The goal of replication policies is simple - to define a target for objects to be copied to.
+
+When applied to a bucketclass, the policy will apply to all future bucket claims that'll utilize the bucketclass.
+
+In general, a replication policy is a JSON-compliant string which defines an array containing at least one rule  -
+  - Each rule is an object containing a `rule_id`, a `destination bucket`, and an optional `filter` key that contains a `prefix` field.
+  - When a filter with prefix is provided - only objects keys that match the prefix will be replicated
+
+The main mechanism behind bucket replication lists all objects across the two buckets, compares the differences, and copies the missing objects from the source to the target bucket. Log-based optimizations utilize AWS S3 server access logging or Azure Monitor to optimize the replication process by copying objects that have been created or modified since the feature was turned on - effectively allowing users to get up to speed with up-to-date recent objects, as the rest replicate in the background with the classic method.
+
+## Replication Policy Parameters
+As stated above, a replication policy is a JSON-compliant array of rules (examples are provided at the bottom of this section)
+  - Each rule is an object that contains the following keys:
+    - `rule_id` - which identifies the rule
+    - `destination_bucket` - which dictates the target NooBaa buckets that the objects will be copied to
+    - (optional) `{"filter": {"prefix": <>}}` - if the user wishes to filter the objects that are replicated, the value of this field can be set to a prefix string
+    - (optional, log-based optimization, see below) `sync_deletions` - can be set to a boolean value to indicate whether deletions should be replicated
+    - (optional, log-based optimization, see below) `sync_versions` - can be set to a boolean value to indicate whether object versions should be replicated
```

Comment:
What do I see below? the example? (it is not clear)
Can you refer to a link for more information about `sync_deletions` and `sync_versions`?

Reply:
It's better.